package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "render-gateway",
-  "version": "1.3.0",
+  "version": "2.0.0",
```

Comment:
**note:** Updated to 2 since this is a breaking change to the types and feature set.